configs/app/js/main.js

```diff
@@ -643,6 +644,16 @@ class Layout extends Component {
                                     onSecondColorChange={this.onChange.bind(this, 'batteryLowColor')}
                                 />
                             )}
+                            {this.isEnabled(['24']) && (
```

Comment:
Minor: are you able to test if it's an android device here?

Reply:
Yep, tested on my android device, all works fine.